### PR TITLE
emby: Drop package and module and refer to jellyfin

### PR DESCRIPTION
**Pull request by etu**
###### Motivation for this change
This is a followup from #54833 where it was suggested to drop emby in favor of jellyfin.

This fixes #54833

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by alyssais — APPROVED** [review]
LGTM as long as everybody else involved is happy.

**Review by worldofpeace — APPROVED** [review]
No problems here. We're inclined to maintain things that we feasibly can.

Though I think for people switching that we should link to their docs
about [database migration](https://jellyfin.readthedocs.io/en/latest/administrator-docs/migrate-from-emby/) (or in general) as migration isn't really possible.

File: pkgs/top-level/all-packages.nix
Comment (worldofpeace) [inline pkgs/top-level/all-packages.nix]: This should probably be in `aliases.nix`.

Comment (aanderse) [reply pkgs/top-level/all-packages.nix]: The concern I have with using an alias instead of throw is that the two are not compatible. We don't want to potentially mangle our users data without them realizing it.

Comment (worldofpeace) [reply pkgs/top-level/all-packages.nix]: Oh no, we can put `throw` in `aliases.nix`.

**Review by aanderse — APPROVED** [review]
I believe that Emby has paid features which differentiate it from Jellyfin. It is possible that some of our users are Emby customers and want to remain that way.

I support removing Emby from master because the new proprietary version of Emby would require a new package and maintainer. I don't see anyone volunteering to do so. *IF* someone wants to package and maintain the new proprietary Emby then they are free to do so in a follow up PR.

Always better to promote free software alternatives :+1:

File: pkgs/top-level/all-packages.nix
Comment (etu) [reply pkgs/top-level/all-packages.nix]: @worldofpeace Good point, moved the throw to aliases.

**Comment by etu** [comment]
@worldofpeace I've added that link to the release notes as well

File: pkgs/top-level/all-packages.nix
Comment (aanderse) [reply pkgs/top-level/all-packages.nix]: @worldofpeace sorry I thought you meant to *not* use a throw, but just alias `emby` as `jellyfin`. In which case ignore my comment :smile:

**Comment by worldofpeace** [comment]
Thanks @etu 

The change to the manual looks like this rendered
Screenshot from 2019-05-01 11 59 06

**Pull request merged**
